apple/RNSVGRenderable.m

```diff
@@ -202,18 +202,23 @@ - (void)renderTo:(CGContextRef)context rect:(CGRect)rect
         CGSize boundsSize = bounds.size;
         CGFloat height = boundsSize.height;
         CGFloat width = boundsSize.width;
+        CGFloat scale = [[UIScreen mainScreen] scale];
```

Comment:
Since we want to support `macOS`, you should not use `UIScreen`, but rather add the option for both platforms analogous to how it is done e.g. with `UIColor`, so just add new define `#define RNSVGScreen UIScreen` here: https://github.com/react-native-svg/react-native-svg/blob/39c8332f4dac8acfe1c60338ad4a8f8231538c91/apple/RNSVGUIKit.h#L12 and `#define RNSVGScreen NSScreen` here: https://github.com/react-native-svg/react-native-svg/blob/39c8332f4dac8acfe1c60338ad4a8f8231538c91/apple/RNSVGUIKit.h#L20 and then use `RNSVGScreen` instead of `UIScreen` in your PR.